minimal/docker-compose.yml

```diff
@@ -53,4 +53,13 @@ services:
     volumes_from:
       - {{ project_name }}_web
     ports:
-      - "0.0.0.0:80:80"
\ No newline at end of file
+      - "0.0.0.0:80:80"
+
+  {{ project_name }}_certbot:
+    restart: always
+    build: ./compose/certbot/production
+    depends_on:
+      - {{ project_name }}_web
+    volumes_from:
```

Comment:
Ah, ok. What you're saying is that rather than have `media` and `staticfiles` in web container, you're moving them to nginx container. If that's the case, then correct, certbot container won't need access to web container, only nginx container. So it would be something like:

```smal_nginx:
    restart: always
    build: ./compose/nginx/staging
    depends_on:
      - smal_web
    volumes:
      - ./media:/code/media
      - ./staticfiles:/code/staticfiles
    ports:
      - "0.0.0.0:8008:80"

smal_certbot:
    restart: always
    build: ./compose/certbot
    depends_on:
        - smal_nginx
    volumes_from:
        - smal_nginx
```